tests/test_config.py

```diff
@@ -4,20 +4,37 @@
 
 """Test config module."""
 
+from operator import itemgetter
+from unittest import mock
+
+import pytest
+
 from mots.config import (
-    calculate_hashes,
     FileConfig,
+    calculate_hashes,
+    clean,
     reference_anchor_for_module,
 )
 from mots.directory import Directory
 
 
+@pytest.fixture
+def test_data():
```

Comment:
Can we make this more specific, like `test_bmo_user_data` or something that more clearly indicates what this data is?